cloud_connectors/mqtt_connector/README.md

```diff
@@ -18,19 +16,75 @@ This section outlines how to use this cloud connector with Azure resources.
+
+* [Azure IoT CLI Extension](https://github.com/Azure/azure-iot-cli-extension)
+
+* [Azure Functions Core Tools](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Cportal%2Cv2%2Cbash&pivots=programming-language-csharp)
+
+#### Azure Resource Group Role-Based Access Control
+
+You will need to be an Owner for your Azure resource group to deploy Azure resources using the scripts. Please see [Azure built-in roles](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles) for more details.
+
+### Deploy Azure Digital Twins
+
+In your Azure Digital Twins resource, you will also need to create digital twin instances. This repository contains [DTDL samples](../resources/sample_dtdl/) to get started.
+
+>NOTE: The deployment scripts are compatible with the [`mixed` sample for Ibeji](https://github.com/eclipse-ibeji/ibeji/tree/main/samples/mixed) and the smart trailer sample from this repository. These scripts will will upload these sample models and create instances of them for you. Modification of the Cloud Digital Twin or setup scripts may be required for other scenarios.
+
+#### Automated Deployment
+
+1. Sign in with Azure CLI. Follow the prompts after entering the following command.
+
+    ```shell
+    az login --use-device-code
+    ```
+
+1. Deploy Azure Digital Twins to your resource group. Use the following to deploy the setup for the Ibeji mixed sample:
+
+    ```shell
+    cd {repo-root}/cloud_connectors/azure/scripts
+    chmod u+x digital_twins_setup.sh
+    ./digital_twins_setup.sh -r {myRG} -l {region} -d {myADT}
+    ```
+
+    Or use the following to deploy the setup for the smart trailer sample:
 
-In your Azure Digital Twins resource, you will also need to create digital twin instances. This repository contains [DTDL samples](../sample-dtdl/) to get started.
+    ```shell
+    cd {repo-root}/cloud_connectors/azure/scripts
+    chmod +x digital_twins_setup_smart_trailer.sh
+    ./digital_twins_setup_smart_trailer.sh -r {myRG} -l {region} -d {myADT}
+    ```
+
+If you experience permission or deployment errors, try running the script again as sometimes it takes a while for some dependencies to be fully deployed. If you use the same name or identifier for each Azure resource, the script will not create additional copies of that Azure resource.
+
+#### Manual Deployment
+
+If you have successfully ran the `digital_twins_setup.sh`, you do not need to follow this section.
+
+The steps below will guide you on manually deploying the Azure Digital Twins resource to your resource group, and creating your Azure Digital Twins instances.
+
+1. Set up your [Azure Digital Twin Instance](https://learn.microsoft.com/en-us/azure/digital-twins/quickstart-azure-digital-twins-explorer#set-up-azure-digital-twins).
+
+    If you wish to use the default mappings in this repository, create the following instances:
+
+    * vehicle
+    * hvac
```

Comment:
when you create these through the azure digital twins interface these are the names you should use, so it should be as-is. @jorchiu do you have anything to add since you originally wrote these docs?

Reply:
You're correct @wilyle. If you change the names of the instances, you will need to change the `instance_id` field in the mock_mapping_service_config (https://github.com/eclipse-ibeji/freyja/blob/main/mocks/mock_mapping_service/res/mock_mapping_config.default.json).

Reply:
in that case this should stay as written